docs/_integration-with-existing-apps-java.md

```diff
@@ -12,7 +12,7 @@ The keys to integrating React Native components into your Android application ar
 
 ## Prerequisites
 
-Follow the React Native CLI Quickstart in the [environment setup guide](environment-setup) to configure your development environment for building React Native apps for Android.
+Follow the guide on [setting up your development environment](set-up-your-environment) and using [React Native without a framework](get-started-without-a-framework) to configure your development environment for building React Native apps for Android.
```

Comment:
I don't think we need to add the second link here no? As the sentence says `to configure your development environment for building React Native apps for Android.`